Doc/source/designspaceLib/readme.rst

```diff
@@ -297,6 +297,8 @@ RuleDescriptor object
 -  Each condition is a dict with ``name``, ``minimum`` and ``maximum`` keys.
 -  ``subs``: list of substitutions
 -  Each substitution is stored as tuples of glyphnames, e.g. ("a", "a.alt").
+-  Note: By default, rules are applied *before* text shaping/OpenType layout. See 
```

Comment:
It's not applied "before text shaping", it is still part of that process, only it occurs at the beginning of it, before the rest of GSUB substitions.

https://docs.microsoft.com/en-us/typography/opentype/spec/features_pt#-tag-rvrn

> should be processed early in GSUB processing, before application of the localized forms feature or features related to shaping of complex scripts or discretionary typographic effects.

Maybe just say that by default they are encoded using `rvrn` (which is applied before all others), unless processing="last" is set. In the latter case, it uses `rclt` (just an arbitrary GSUB feature really that is always on basically) which is applied last because we add the rules lookup at the end of the lookup list and rclt follows the specified order.